plugins/org.locationtech.udig.project.tests/META-INF/MANIFEST.MF

```diff
@@ -24,3 +24,5 @@ Require-Bundle: org.junit;bundle-version="[4.0.0,5.0.0)",
  org.easymock
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Import-Package: org.locationtech.udig.project.ui,
```

Comment:
IMO we should keep UI contributions seperated from core tests. What do you think about creating a new bundle org.locationtech.udig.project.ui.tests which has a dependency to org.locationtech.udig.project.ui and provides Eclipse menue and command contributions.

It should not be part of a standard eclipse feature to avoid getting it into product. On the other hand it might be worth to have it in sdk

Reply:
actually there is plugin org.locationtech.udig.project.tests.ui. 
Maybe I can use this for defining the menu and command contribution since it already contains other eclipse UI elements.
What do you think @fgdrf ?